### ISSUE TITLE
[A11y] Home - Items unreachable via tab key

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2188]
### Operating System

Windows

### Browser

Firefox

### Affected page

Homepage

### Admin user?

No

### Issue Description

Various items are not in the tab sequence. That is to say, they're not focusable via the tab key (or if they are, I'm not seeing how to get to them).

- The "Sign Up" link at the top, next to the "LOG IN" button, seems to have an explicit tab index of -1, which means it cannot be reached except with JavaScript
- The collections' thumbnails also have a tab index of -1

### Suggested Fix

I think just removing the tab index attribute would fix this.

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

- Using the tab key alone, make sure you can navigate to "Sign In" and activate it (via the spacebar or enter key)
- Using the tab key alone, make sure you can navigate to collections' featured images and activate them (these are links, so only the enter key needs to work)
